packages/liveblocks-react/src/__tests__/useThreads.test.tsx

```diff
@@ -450,8 +450,8 @@ describe("useThreads", () => {
     });
     const subscriptions = [
       dummySubscriptionData({ subjectId: bluePinnedThread.id }),
-      dummySubscriptionData({ subjectId: blueUnpinnedThread.id }),
-      dummySubscriptionData({ subjectId: redPinnedThread.id }),
+      // dummySubscriptionData({ subjectId: blueUnpinnedThread.id }),
+      // dummySubscriptionData({ subjectId: redPinnedThread.id }),
```

Comment:
Should we remove them?

Reply:
It helps to understand which threads are subscribed to and which ones are not.
But I updated the text so it's clearer.